pkgs/servers/plex/raw.nix

```diff
@@ -12,16 +12,16 @@
 # server, and the FHS userenv and corresponding NixOS module should
 # automatically pick up the changes.
 stdenv.mkDerivation rec {
-  version = "1.25.2.5319-c43dc0277";
+  version = "1.25.3.5409-f11334058";
   pname = "plexmediaserver";
 
   # Fetch the source
   src = if stdenv.hostPlatform.system == "aarch64-linux" then fetchurl {
     url = "https://downloads.plex.tv/plex-media-server-new/${version}/debian/plexmediaserver_${version}_arm64.deb";
-    sha256 = "09kkkyli5fygyvlzqd46jzi0y4jp0a24d92ayvfm95gm3fcxl73x";
+    sha256 = "sha256-Q0768UxZXSuUm26Ro2XIrw8qoQEWCU23P7EBtbiIa0c=";
   } else fetchurl {
     url = "https://downloads.plex.tv/plex-media-server-new/${version}/debian/plexmediaserver_${version}_amd64.deb";
-    sha256 = "17whd724sjblqxz6d79jb6hrqvkgg5mbh3bh1lr9b8sswa3pxb07";
+    sha256 = "sha256-TaOvXVeo385vqtv/dYvmQQx8c1K/EMw4MTDucg511Cs=";
```

Comment:
Technically your hashes works. But I think it isn't best practice to mention the hash type twice.
We could replace the "sha256" by "hash", so it would look like this:
```nix
hash = "sha256-Q0768UxZXSuUm26Ro2XIrw8qoQEWCU23P7EBtbiIa0c=";
```
The alternative would be, to stay with the sha256 format we used before.
nix-prefetch-url returns it by default.
For example:
```bash
nix-prefetch-url https://downloads.plex.tv/plex-media-server-new/1.25.3.5409-f11334058/debian/plexmediaserver_1.25.3.5409-f11334058_amd64.deb
path is '/nix/store/x9h59mix9md08p6yz5hf3lvrm0z4h67n-plexmediaserver_1.25.3.5409-f11334058_amd64.deb'
0aylfl775vih64wcq45za9rpq321ws5pbzyvm9pwxpx8axfsz8sd
```

Conclusion: technically your PR works perfectly fine! But I think it would be nicer to change the hash format.